spec/bookshelf-spec.ts

```diff
@@ -616,6 +616,48 @@ describe('Bookshelf links', () => {
     expect(_.matches(expected)(result)).toBe(true);
   });
 
+  it('should not add pagination links if collection is empty', () => {
+    let limit: number = 10;
+    let offset: number = 40;
```

Comment:
Sorry, I was looking at the wrong test. In this case, I'm checking to see if the collection is empty (`total = 0`), so the `offset` and `limit` are irrelevant.